src/loader-components/PropsTable.module.scss

```diff
@@ -1,28 +1,33 @@
-@import '../mdx/MarkdownTags.module';
 @import './clinical-lowlight-theme/PropsTable.module';
 @import './orion-fusion-theme/PropsTable.module';
 
 :local {
-  .required {
-    color: #d53040;
+  .bold {
+    font-weight: 600;
   }
 
-  .props-tr {
-    vertical-align: top;
+  .table {
+    border-collapse: collapse;
+    border-spacing: 0;
+    display: block;
+    margin-bottom: 16px;
```

Comment:
You could use rems here if you're worried about adding font scaling to dev-site in the future.

Reply:
updated